hdt-lib/include/HDTVocabulary.hpp

```diff
@@ -33,12 +33,13 @@
 #define HDT_HDTVOCABULARY_HPP_
 
 #include <string>
+#include <HDTVersion.hpp>
 
 namespace hdt {
 namespace HDTVocabulary {
 	// Base
 	const std::string HDT_BASE = "<http://purl.org/HDT/hdt#";
-	const std::string HDT_CONTAINER = HDT_BASE+"HDTv1>";
+	const std::string HDT_CONTAINER = HDT_BASE+"HDTv"+HDT_VERSION+">";
```

Comment:
Hmm, I didn't do that for two reasons: the breaking changes, and I'd had to write a function that parses this URI to compare, but didn't know where to put it.
But what you're saying is also my preference, just more work.